Getting started example fails to compile with latest version

The [Getting Started](https://github.com/tonybaloney/CSnakes/blob/07e4e2171d88deac63cf3974e7bef19721e5a915/docs/getting-started.md#constructing-a-python-environment-from-c) documentation shows this code for getting started

```c#
using CSnakes.Runtime;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

var builder = Host.CreateDefaultBuilder(args)
    .ConfigureServices(services =>
    {
        var home = Path.Join(Environment.CurrentDirectory, "."); /* Path to your Python modules */
        services
            .WithPython()
            .WithHome(home)
            .FromRedistributable(); // Download Python 3.12 and store it locally
    });

var app = builder.Build();

var env = app.Services.GetRequiredService<IPythonEnvironment>();
```

but this fails to compile with [the latest version ](https://github.com/tonybaloney/CSnakes/releases/tag/v1.0.30) due to a call ambiguity error:

    Program.cs(16,14): error CS0121: The call is ambiguous between the following methods or properties: 'ServiceCollectionExtensions.FromRedistributable(IPythonEnvironmentBuilder, RedistributablePythonVersion, bool, bool, int)' and 'ServiceCollectionExtensions.FromRedistributable(IPythonEnvironmentBuilder, string, bool, bool, int)'

The workaround for now is to change it as follows:

```diff
diff --git a/Program.cs b/Program.cs
index b3c9e86..34465df 100644
--- a/Program.cs
+++ b/Program.cs
@@ -1,4 +1,5 @@
 using CSnakes.Runtime;
+using CSnakes.Runtime.Locators;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using System;
@@ -13,7 +14,7 @@ var builder = Host.CreateDefaultBuilder(args)
         services
             .WithPython()
             .WithHome(home)
-            .FromRedistributable(); // Download Python 3.12 and store it locally
+            .FromRedistributable(RedistributablePythonVersion.Python3_12); // Download Python 3.12 and store it locally
     });
 
 var app = builder.Build();
```

--- a/src/CSnakes.Runtime/ServiceCollectionExtensions.cs
+++ b/src/CSnakes.Runtime/ServiceCollectionExtensions.cs
@@ -181,6 +181,26 @@ public static partial class ServiceCollectionExtensions
 
     /// <summary>
     /// Simplest option for getting started with CSnakes.
+    /// Downloads and installs the redistributable version of Python 3.12 from GitHub and stores it in %APP_DATA%/csnakes.
+    /// </summary>
+    /// <param name="builder">The <see cref="IPythonEnvironmentBuilder"/> to add the locator to.</param>
+    /// <param name="debug">Whether to use the debug version of Python.</param>
+    /// <param name="timeout">Timeout in seconds for the download and installation process.</param>
+    /// <returns></returns>
+    public static IPythonEnvironmentBuilder FromRedistributable(this IPythonEnvironmentBuilder builder, bool debug = false, int timeout = 360)
+    {
+        builder.Services.AddSingleton<PythonLocator>(
+            sp =>
+            {
+                var logger = sp.GetRequiredService<ILogger<RedistributableLocator>>();
+                return new RedistributableLocator(logger, RedistributablePythonVersion.Python3_12, timeout, debug);
+            }
+        );
+        return builder;
+    }
+
+    /// <summary>
+    /// Simplest option for getting started with CSnakes.
     /// Downloads and installs the redistributable version of Python from GitHub and stores it in %APP_DATA%/csnakes.
     /// </summary>
     /// <param name="builder">The <see cref="IPythonEnvironmentBuilder"/> to add the locator to.</param>
@@ -189,7 +209,7 @@ public static partial class ServiceCollectionExtensions
     /// <param name="freeThreaded">Free Threaded Python (3.13+ only)</param>
     /// <param name="timeout">Timeout in seconds for the download and installation process.</param>
     /// <returns></returns>
-    public static IPythonEnvironmentBuilder FromRedistributable(this IPythonEnvironmentBuilder builder, RedistributablePythonVersion version = RedistributablePythonVersion.Python3_12, bool debug = false, bool freeThreaded = false, int timeout = 360)
+    public static IPythonEnvironmentBuilder FromRedistributable(this IPythonEnvironmentBuilder builder, RedistributablePythonVersion version, bool debug = false, bool freeThreaded = false, int timeout = 360)
     {
         builder.Services.AddSingleton<PythonLocator>(
             sp =>
@@ -211,7 +231,7 @@ public static partial class ServiceCollectionExtensions
     /// <param name="freeThreaded">Free Threaded Python (3.13+ only)</param>
     /// <param name="timeout">Timeout in seconds for the download and installation process.</param>
     /// <returns></returns>
-    public static IPythonEnvironmentBuilder FromRedistributable(this IPythonEnvironmentBuilder builder, string version = "3.12", bool debug = false, bool freeThreaded = false, int timeout = 360)
+    public static IPythonEnvironmentBuilder FromRedistributable(this IPythonEnvironmentBuilder builder, string version, bool debug = false, bool freeThreaded = false, int timeout = 360)
     {
         RedistributablePythonVersion versionEnum = version switch
         {

